packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md

```diff
@@ -210,8 +210,10 @@ For more information on implementing inline hooks, see the documentation for spe
 
 ## Troubleshooting
 
-The [Okta System Log](/docs/reference/api/system-log/) captures events related to inline hook setup and execution, which you can use to troubleshoot your implementation. To see descriptions of the relevant event types, query the Event Types catalog with the query parameter `inline_hook`:
+A hook preview feature is available for the following hooks: SAML, telephony, token, and registration inline hooks. Use this feature to view a sample request body and evaluate and troubleshoot the response from your external service. See [Troubleshoot hook implementations](/docs/guides/common-hook-set-up-steps/nodejs/main/#troubleshoot-hook-implementations) and [Preview an inline hook](https://help.okta.com/okta_help.htm?type=oie&id=ext-preview-inline-hooks).
 
-<https://developer.okta.com/docs/reference/api/event-types/?q=inline_hook>
+The [Okta System Log](/docs/reference/api/system-log/) also captures events related to inline hook setup and execution, which you can use to troubleshoot your implementation. To see descriptions of the relevant event types, query the Event Types catalog with the query parameter `inline_hook`:
+
+<https://developer.okta.com/docs/reference/api/event-types/>
```

Comment:
Not sure if you're using this format for a reason? Or can we use the regular format:
```
See [Event Types inline hook query](https://developer.okta.com/docs/reference/api/event-types/?q=inline_hook)
or 
See [Event Types inline hook query](/docs/reference/api/event-types/?q=inline_hook)
```

Reply:
Historically left in there, I assumed, to highlight the query parameter. I don't think this direct link is working, though (it isn't after I checked), so, I'll remove this bit. I thought I fixed it, but I think that was for Event Types.

Reply:
Updated. Thanks.